README.md

```diff
@@ -355,6 +356,62 @@ writable.
 
 Always prefer `ReadonlyArray` over a regular `Array` unless it must be possible to modify the Array.
 
+#### Preferring Interfaces Over Intersections
+
+`interface` are a little more powerful than the corresponding `type` declaration.
```

Comment:
This is incorrect; they have 90% overlap, and besides that they have their own use cases and powers.